src/SDK/MonitoringApplianceEmulator/Controls/TablePropertyControl.xaml

```diff
@@ -17,9 +17,10 @@
                 <Setter Property="Template">
                     <Setter.Value>
                         <ControlTemplate TargetType="{x:Type DataGridCell}"  >
-                            <TextBlock Text="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=Content.Text}" 
-                                       ToolTip="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=Content.Text}"
-                                       TextTrimming="CharacterEllipsis" Margin="12"/>
+                            <local:HypertextBlock Hypertext="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=Content.Text}"
+                                                   ToolTip="{Binding RelativeSource={RelativeSource Self}, Path=Hypertext}"
```

Comment:
>ToolTip="{Binding RelativeSource={RelativeSource Self}, Path=Hypertext}" [](start = 51, length = 72)

will this show the links?

Reply:
currently with the HRF.I added the improvement task to include this fix as well.

---
In reply to: [234513231](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/25#discussion_r234513231) [](ancestors = 234513231)